components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -424,6 +424,22 @@ - (void)setCharacterCountViewMode:(UITextFieldViewMode)characterCountViewMode {
   }
 }
 
+- (void)setDisabledColor:(UIColor *)disabledColor {
```

Comment:
Please propose in weekly meeting. If it's accepted as a team best practice, then let's do it.